Resources/views/Collector/db.html.twig

```diff
@@ -186,55 +162,83 @@
 
     <h2>Database Connections</h2>
 
-    {% if collector.connections %}
-        {% include 'WebProfilerBundle:Profiler:table.html.twig' with {data: collector.connections} only %}
+    {% if not collector.connections %}
+        <div class="empty">
+            <p>There are no configured database connections.</p>
+        </div>
     {% else %}
-        <p>
-            <em>No connections.</em>
-        </p>
+        {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.connections, labels: ['Name', 'Service'] }, with_context = false ) }}
```

Comment:
please use the ``@WebProfiler/`` naming instead to reference WebProfilerBundle templates